build_tools/aws-sdk-code-generator/templates/spec/endpoint_provider_spec_class.mustache

```diff
@@ -11,7 +11,7 @@ module {{module_name}}
     subject { {{module_name}}::EndpointProvider.new }
 
     {{#endpoint_tests}}
-    context '{{documentation}}' do
+    context "{{{documentation}}}" do
```

Comment:
Do we expect special symbols in the context description?

Reply:
Yeah, this fixes an issue currently.